<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginInfoDialog</class>
 <widget class="QDialog" name="PluginInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BGS Data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabRecords">
      <attribute name="title">
       <string>Records</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="BSPluginInfo::PluginRecordView" name="pluginRecordView" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabArchives">
      <attribute name="title">
       <string>Archives</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTabWidget" name="tabArchivesTabs">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabConflictsGeneral">
          <attribute name="title">
           <string>Conflicts</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1,0,1,0,1">
           <item>
            <layout class="QVBoxLayout" name="winningRoot">
             <item>
              <layout class="QHBoxLayout" name="winningHeader" stretch="1,0,0">
               <item>
                <widget class="QToolButton" name="winningExpander">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="toolTip">
                  <string>Files that exist in other mods but are overwritten by this mod</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border: none;
text-align: left;</string>
                 </property>
                 <property name="text">
                  <string>Winning file conflicts:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="MOBase::LineEditClear" name="winningLineEdit">
                 <property name="placeholderText">
                  <string>Filter</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="winningCount">
                 <property name="frameShadow">
                  <enum>QFrame::Sunken</enum>
                 </property>
                 <property name="lineWidth">
                  <number>1</number>
                 </property>
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTreeView" name="winningTree">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="toolTip">
              <string>Files that exist in other mods but are overwritten by this mod</string>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="textElideMode">
              <enum>Qt::ElideLeft</enum>
             </property>
             <property name="uniformRowHeights">
              <bool>true</bool>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="losingRoot">
             <item>
              <layout class="QHBoxLayout" name="losingHeader" stretch="1,0,0">
               <item>
                <widget class="QToolButton" name="losingExpander">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="toolTip">
                  <string>Files that are unused because they are overwritten by other mods</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border: none;
text-align: left;</string>
                 </property>
                 <property name="text">
                  <string>Losing file conflicts:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="MOBase::LineEditClear" name="losingLineEdit">
                 <property name="placeholderText">
                  <string>Filter</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="losingCount">
                 <property name="frameShadow">
                  <enum>QFrame::Sunken</enum>
                 </property>
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTreeView" name="losingTree">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="toolTip">
              <string>Files that are unused because they are overwritten by other mods</string>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="textElideMode">
              <enum>Qt::ElideLeft</enum>
             </property>
             <property name="uniformRowHeights">
              <bool>true</bool>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="noConflictRoot">
             <item>
              <layout class="QHBoxLayout" name="noConflictHeader" stretch="1,0,0">
               <item>
                <widget class="QToolButton" name="noConflictExpander">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="toolTip">
                  <string>Files that have no conflicts</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border: none;
text-align: left;</string>
                 </property>
                 <property name="text">
                  <string>The following files have no conflicts:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="MOBase::LineEditClear" name="noConflictLineEdit">
                 <property name="placeholderText">
                  <string>Filter</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="noConflictCount">
                 <property name="frameShadow">
                  <enum>QFrame::Sunken</enum>
                 </property>
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTreeView" name="noConflictTree">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="toolTip">
              <string>Files that have no conflicts</string>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="textElideMode">
              <enum>Qt::ElideLeft</enum>
             </property>
             <property name="uniformRowHeights">
              <bool>true</bool>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabArchivesTree">
          <attribute name="title">
           <string>Tree</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0">
             <item>
              <widget class="QLabel" name="currentArchiveLabel">
               <property name="text">
                <string notr="true"/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="previousArchiveButton">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../../../modorganizer/src/resources.qrc">
                 <normaloff>:/MO/gui/previous</normaloff>:/MO/gui/previous</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="nextArchiveButton">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../../../modorganizer/src/resources.qrc">
                 <normaloff>:/MO/gui/next</normaloff>:/MO/gui/next</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QStackedWidget" name="archivesTreeStack"/>
           </item>
           <item>
            <widget class="MOBase::LineEditClear" name="archiveFilterEdit">
             <property name="baseSize">
              <size>
               <width>220</width>
               <height>0</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>Filter</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="previousFile">
       <property name="text">
        <string>Previous</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextFile">
       <property name="text">
        <string>Next</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="close">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MOBase::LineEditClear</class>
   <extends>QLineEdit</extends>
   <header>lineeditclear.h</header>
  </customwidget>
  <customwidget>
   <class>BSPluginInfo::PluginRecordView</class>
   <extends>QWidget</extends>
   <header>bsplugininfo/pluginrecordview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../modorganizer/src/resources.qrc"/>
 </resources>
 <connections/>
</ui>
